tests/integ/sagemaker/lineage/test_artifact.py

```diff
@@ -120,6 +127,33 @@ def validate():
     retry(validate, num_attempts=3)
 
 
+def test_downstream_trials_v2(trial_associated_artifact, trial_obj, sagemaker_session):
+    trials = trial_associated_artifact.downstream_trials_v2()
+    assert len(trials) == 1
+    assert trial_obj.trial_name in trials
+
+
+def test_upstream_trials(upstream_trial_associated_artifact, trial_obj, sagemaker_session):
+    trials = upstream_trial_associated_artifact.upstream_trials()
+    assert len(trials) == 1
+    assert trial_obj.trial_name in trials
+
+
+def test_upstream_trials_v2(trial_associated_artifact, trial_obj):
+    def validate():
+        for i in range(3):
```

Comment:
Do we need this manual retry if we're already wrapping this function in `retry()` ?

Reply:
> Do we need this manual retry if we're already wrapping this function in `retry()` ?

this test case is redundant, removed the test case